### ISSUE TITLE
Oil leak from valve cover seal

### ISSUE THREAD

**Issue by uncheckederror** [uncheckederror/toyotapickup#2]
There appears to be an oil leak coming from the rear of the valve cover. More investigation is required.

[Here's a valve cover seal replacement kit.](https://22reperformance.com/22re-gaskets-and-seals/valve-cover-gasket-kits)

**Comment by uncheckederror** [comment]
I have removed the valve cover and extracted the gasket.

Removing the old gasket around the PCV valve proved to be extremely difficult and the old PCV valve was irreparably damaged in the process.

Here's the engine with the valve cover removed.

![image](https://user-images.githubusercontent.com/11726956/47261208-19960f80-d47f-11e8-8829-e82c13e374ba.png)

In the two images below you can see the cracking and failures int he old valve cover gasket.

![image](https://user-images.githubusercontent.com/11726956/47261220-41857300-d47f-11e8-9f72-55d920e66cad.png)

![image](https://user-images.githubusercontent.com/11726956/47261225-4c400800-d47f-11e8-950c-d2bdd9da7ccd.png)

Next steps are to buy a new PCV valve, some silicon sealant for the half-moon gaskets, and complete the gasket replacement process.

**Comment by uncheckederror** [comment]
I have replace the valve cover seal. I have also replaced the PCV valve, it's seal, and the the valve cover stud seals.

![image](https://user-images.githubusercontent.com/11726956/47263087-131f8c00-d4af-11e8-9ab9-761943325cf2.png)

![image](https://user-images.githubusercontent.com/11726956/47263088-203c7b00-d4af-11e8-8efc-ce278ec97fde.png)

I briefly cleaned the valve cover and all of the bolts and mounting hardware that attach to the valve cover. 

I used an air compressor set to 80 psi to blow out the vacuum lines.

The rubber half moons at either end of the valve cover were installed using silicon gasket material, applied in a thick bead, on the underside of the moon. Then I pushed them out from the inside of the engine so that they were pressed up against the inside of the engine wall.

Prior to reattaching the valve cover I cleaned the mating surfaces, on both the head and the valve cover, by running paper towels along them until they appeared relatively clean.

I do not have a torque wrench so I have torqued all of the bolts on this valve cover to tight given the minimum effort that I can apply using my standard size ratchet.

After reassembling the engine with the new gaskets and cleaned parts I started up the truck and let it run for about 10 minutes.

After it reached its normal idle I revved the engine using the throttle cable on the engine block a couple of times. 

The engine sounded smoother than it had before and didn't shake at all.

There is a rapid ticking noise that I believe is from the fuel injectors actuating. 

There is also a deep whooshing noise when the engine is revved that I believe is coming from the air intake box. I'm not sure what's causing this noise, and I'm starting to think that it might be the intended behavior of the engine. But I think it's worth investigating this issue further.
